payload/bin/grepCIDR

```diff
@@ -67,10 +67,10 @@
 umask 137
 
 while [[ ($# -gt 1) && (${1:1:1} = '-') ]] ; do
-GREP_ARGS="${GREP_ARGS} ${1}"
-shift
-done
+GREP_ARGS="${GREP_ARGS} ${1}";
+shift ;
+done ;
```

Comment:
😕 line 70 `GREP_ARGS="${GREP_ARGS} ${1}";` is already quoted?!? why do you think the double quotes `"` are not enough here? and even if it did split this loop re-adds the space.

Please carefully double check, you seem confused; and it is confusing me.

I think it would be fine to just address the lack of documentation:

```suggestion
# Accumulate grep arguments
GREP_ARGS="${GREP_ARGS} ${1}";
shift ;
done ;
```

Am I mistaken somehow? why or why not?